src/ui/components/ListaProductos/filter/DividerGroup.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Divider } from '@mui/material'
+import styled from '@emotion/styled';
+
+const StyledDivider = styled(Divider)(({ theme }) => ({
+    width:'194px',
+    height:'1px',
+    zIndex:1,
+    position:'absolute',
+    left:'19px',
+    backgroundColor:theme.palette.filter.divider,
+    
+  }));
+
+function DividerGroup(props) {
```

Comment:
please use the definition of a function like this
const dividerGroup = () => {
}